extend/registration/index.md

```diff
@@ -1,70 +1,101 @@
 ---
 title: Component Registration
 permalink: /extend/registration/
+redirect_from: /extend/registration/checklist/
 ---
 
 * TOC
 {:toc}
 
-## Introduction
-As described in the [architecture overview](/overview/), Keboola Connection (KBC) consists of many different components. 
-Only those components that are registered in our **Component List** are generally available in KBC. 
+As described in the [architecture overview](/overview/), Keboola Connection (KBC) consists of many different components.
+Only those components that are registered in our **Component List** are generally available in KBC.
 The list is provided by our [Storage Component API](http://docs.keboola.apiary.io/#) in the dedicated [Components section](http://docs.keboola.apiary.io/#reference/components).
+The list of Components is manages using [Keboola Developer](https://apps.keboola.com/) portal.
 
-While a [Custom Science extension](/extend/custom-science/) requires registration only when offered to all KBC users, 
+While a [Custom Science extension](/extend/custom-science/) requires registration only when offered to all KBC users,
 registering a [Docker extension](/extend/docker/) is mandatory at all times.
 
 That being said, any KBC user can use any registered component, unless
 
 - the KBC user (or their token) has a [limited access to the component](https://help.keboola.com/storage/tokens/),
 - the component itself limits where it can run (in what projects and for which users).

```

Comment:
tady bych možná doplnil, že je ještě možný aplikaci schovat, ale to neznamená, že jí někdo nemůž použít, že jen není nabízená v seznamech